core/src/main/java/org/wildfly/extras/creaper/core/online/OnlineOptions.java

```diff
@@ -389,16 +409,16 @@ public void handle(Callback[] callbacks) throws IOException, UnsupportedCallback
             }
 
             modelControllerClient = (ModelControllerClient) createMethod.invoke(null, // static method
-                    protocolName, host, port, callbackHandler, null, connectionTimeout, saslOptions);
+                    protocolName, host, port, callbackHandler, sslContext, connectionTimeout, saslOptions);
         } catch (NoSuchMethodException e) {
-            if (protocol == ManagementProtocol.HTTP_REMOTING) {
+            if (protocol == ManagementProtocol.HTTP_REMOTING || protocol == ManagementProtocol.HTTPS_REMOTING) {
                 // user asks for WildFly, but the client library is from AS7, this can't work
                 throw new IllegalStateException("The server should be WildFly (either ManagementProtocol.HTTP_REMOTING was used or the '"
```

Comment:
Please change `ManagementProtocol.HTTP_REMOTING` to `ManagementProtocol.HTTP(S)_REMOTING` (or something similar) in the exception message.